db/migrate/20150904181202_miq_groups_add_ldap_role.rb

```diff
@@ -0,0 +1,17 @@
+class MiqGroupsAddLdapRole < ActiveRecord::Migration
+  class MiqUserRole < ActiveRecord::Base
+  end
+
+  class MiqGroup < ActiveRecord::Base
+  end
```

Comment:
@jrafanie the inheritance column thing.

If the table doesn't have a `type` column at the time this migration is defined, it won't have a `type` column at any future time the migration is run.